Resources/Prototypes/_NF/Catalog/Fills/Lockers/chemistry_storage_wallmount.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  id: LockerWallChemistryFilled
+  suffix: Filled
+  parent: LockerWallChemistry
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSyringe
+      - id: BoxBeaker
+        amount: 2
+      - id: BoxPillCanister
+      - id: BoxBottle
+      - id: BoxVial
+      - id: ChemBag
+      - id: ClothingHandsGlovesLatex
+#      - id: ClothingHeadsetMedical # Frontier
+      - id: ClothingMaskSterile
+#      - id: HandLabeler
+      - id: ClothingEyesGlassesChemical # Frontier
+      - id: SheetPlasma1 # Frontier
+        amount: 3
```

Comment:
A better way to go about it would be to use double parent like this:
```
- type: entity
  id: LockerWallChemistryFilled
  parent: [ LockerWallChemistry, LockerChemistryFilled ]
```
Those 3 lines is all you need: removes the need for listing down contents.